package.json

```diff
@@ -44,7 +44,8 @@
     "less-loader": "^6.0.0",
     "lunr": "^2.3.8",
     "moment": "^2.24.0",
-    "postcss-less": "^6.0.0"
+    "postcss-less": "^6.0.0",
+    "write-excel-file": "^1.4.27"
```

Comment:
We've begun committing the package lock file (it's now required for CI and our builds). Could you execute `npm install` and commit the revised package lock file?

Additionally, we should be able to eliminate `exceljs` and `file-saver` from our dependencies with these changes (the license file will also need to be updated).